authui/src/authflowv2/icons/authgear-icon.css

```diff
@@ -0,0 +1,20 @@
+.authgear-icon {
+  @apply w-16 h-16 bg-no-repeat bg-center;
+  background-image: url(./icons/raw/icon-authgear.svg);
```

Comment:
@Ngkaokis Sorry I forgot to mention in the issue:
The logo is not hardcoded and can be configured per project.
Please see `resources/authgear/templates/en/web/__header.html` for the current implementation.
Also, the logo size is adjustable according to the spec, so please make it a css variable.

Reply:
why display two logos if dark theme enabled :derpthink:
```
{{ if .darkThemeEnabled }}
<img class="banner light" src="{{ .light }}">
<img class="banner dark" src="{{ .dark }}">
{{ else }}
<img class="banner" src="{{ .light }}">
{{ end }}

{{ end }}

```

Reply:
@Ngkaokis If dark theme disabled, no matter your device is dark or light theme, it shows the light icon.
If dark theme enabled, depends on your device's theme, it show one icon and hide another one.

Reply:
Do you expect to duplicate the header implementation from v1 to v2?

Reply:
See authui/src/tailwind.css:308